libs/sdk-ui-tests/scenarios/pivotTable/drilling.tsx

```diff
@@ -0,0 +1,33 @@
+// (C) 2007-2019 GoodData Corporation
+import { action } from "@storybook/addon-actions";
+import { scenariosFor } from "../../src";
+import { IPivotTableProps, PivotTable } from "@gooddata/sdk-ui-pivot";
+import { PivotTableWithTwoMeasuresAndThreeRowsAndTwoCols } from "./base";
+import {
+    DepartmentPredicate,
+    ProductPredicate,
+    SalesRepPredicate,
+    AmountMeasurePredicate,
+    WonMeasurePredicate,
+} from "../_infra/predicates";
+
+export default scenariosFor<IPivotTableProps>("PivotTable", PivotTable)
+    .withGroupNames("drilling")
+    .withVisualTestConfig({ screenshotSize: { width: 1200, height: 800 } })
+    .withDefaultTags("vis-config-only", "mock-no-scenario-meta")
+    .addScenario("Pivot table with drill on all row atributes", {
+        ...PivotTableWithTwoMeasuresAndThreeRowsAndTwoCols,
+        drillableItems: [ProductPredicate, DepartmentPredicate, SalesRepPredicate],
+        onDrill: action("onDrill"),
+    })
+    .addScenario("Pivot table with drill on all row attributes and measures", {
```

Comment:
I think you don't need the 'Pivot Table' prefix here. the infra will add automatically

Reply:
You're right, updated (It's correctly put to url and in the tree it is not in the story title but defined by the hierrarchy listing)